tools/packaging/packaging.strings.psd1

```diff
@@ -73,4 +73,50 @@ OsxDistributionTemplate = @'
     <pkg-ref id="com.microsoft.powershell" version="{1}" onConclusion="none">{2}</pkg-ref>
 </installer-gui-script>
 '@
+NuspecTemplate = @'
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>{0}</id>
+        <version>{1}</version>
+        <title>PowerShellRuntime</title>
+        <authors>Powershell</authors>
+        <owners>microsoft,powershell</owners>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>PowerShell runtime for hosting PowerShell</description>
+        <copyright>Copyright (c) Microsoft Corporation. All rights reserved.</copyright>
```

Comment:
I don't see field with MIT license - is it need?

Reply:
That is only required for source code, this is not source code.